app/code/Magento/MediaStorage/Console/Command/ImagesResizeCommand.php

```diff
@@ -28,24 +28,24 @@ class ImagesResizeCommand extends \Symfony\Component\Console\Command\Command
     private $appState;
 
     /**
-     * @var ObjectManagerInterface
+     * @var ProgressBarFactory
      */
-    private $objectManager;
+    private $progressBarFactory;
 
     /**
      * @param State $appState
      * @param ImageResize $resize
-     * @param ObjectManagerInterface $objectManager
+     * @param ProgressBarFactory $progressBarFactory
      */
     public function __construct(
         State $appState,
         ImageResize $resize,
-        ObjectManagerInterface $objectManager
+        ProgressBarFactory $progressBarFactory
     ) {
         parent::__construct();
         $this->resize = $resize;
         $this->appState = $appState;
-        $this->objectManager = $objectManager;
+        $this->progressBarFactory = $progressBarFactory;
```

Comment:
Unfortunately, according to the Contribution Guide, we may not remove constructor params.
The new constructor parameters should be optional in order to avoid backward compatibility issues.
Please check the [Contribution Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/#adding-a-constructor-parameter) (section *Adding a constructor parameter*) for more details.

To preserve the constructor signature:
- Remove type hinting for unused parameters to remove dependency on their type.
- Add `@SuppressWarnings(PHPMD.UnusedFormalParameter)` for unused parameters.

Reply:
Thank you for the review! I updated the changeset, fixed coding style errors and fixed the BIC.